Atlas/backend/atlas/views.py

```diff
@@ -90,3 +98,22 @@ def get_queryset(self):
         user=self.request.user;
         return Cultural_Heritage.objects.filter(user=user)
 
+class cultural_heritage_item_search(generics.ListAPIView):
+    serializer_class = cultural_heritage_serializer
+    permission_classes = []
+
+    def get_queryset(self):
+        query = self.kwargs.get('query')
+        return Cultural_Heritage.objects.annotate(
+           search=SearchVector('tags__name', 'title','description'),
+        ).filter(search=query)
+
+class cultural_heritage_item_search_autocorrect(generics.ListAPIView):
+    permission_classes = []
+    serializer_class = cultural_heritage_serializer
+    pagination_class = None
```

Comment:
I think we shouldnt change the default pagination here. It will be more flexible than just returning 5 items.